src/generate/project.rs

```diff
@@ -175,7 +175,17 @@ history.txt
 name = "{}"
 authors = []
 telemetry = {}
-analysis = ["check_checker"]
+[project.repl.analysis]
```

Comment:
What's your take on flattening a bit the schema here? `repl.analysis`, instead of `project.repl.analysis`?

Reply:
Sure, that makes sense. I will quickly make that change and push.

Reply:
Changed in a569502